api_docs/changelog.md

```diff
@@ -20,6 +20,15 @@ format used by the Zulip server that they are interacting with.
 
 ## Changes in Zulip 9.0
 
+**Feature level 263**
+
+* [`GET /events`](/api/get-events): Added `scheduled_end_time` field
+  to `user_status` object.
+* [`POST /register`](/api/register-queue): Added `scheduled_end_time`
+  field to `user_status` objects.
+* [`POST /users/me/status`](/api/update-status): Added support for new
+  `scheduled_end_time` parameter.
+
```

Comment:
`/api/get-user-status` is missing.

Reply:
Also, we can list it as `[GET /events](/api/get-events), [POST /register](/api/register-queue): description...` -- see other entries in this file for motivation.